New theorems with pi-character (#1807)
pi-base/data: theorems/T000909.md

---
uid: T000909
if:
  and:
  - P000002: true
  - P000174: true

>   - P000244: true

Reviewer: ```suggestion
  - P000174: true
  - P000244: true
  - P000002: true
```
move the non-essential auxiliary condition to the end

then:
  P000028: true
---

> Fix some point $x$; we wish to find a countable local base for $x$. Let $\mathcal{U}$ be a local base for $x$, linearly ordered by inclusion, and $\mathcal{A}$ be a countable local $\pi$-base for $x$. WLOG assume $x$ is not isolated (otherwise $\{\{x\}\}$ is a countable local base for $x$). Therefore, for any $A \in \mathcal{A}$, there is some $a \in A \setminus \{x\}$; by $T_1$, we can choose some $U_A \in \mathcal{U}$ with $U_A \subseteq A \setminus \{a\}$.

Reviewer: ```suggestion
Fix some point $x$; we wish to find a countable local base for $x$. Let $\mathcal{U}$ be a local base for $x$, linearly ordered by inclusion, and $\mathcal{A}$ be a countable local $\pi$-base for $x$. WLOG assume $x$ is not isolated (otherwise $\{\{x\}\}$ is a countable local base for $x$).
Therefore, for any $A \in \mathcal{A}$, there is some $a \in A \setminus \{x\}$.
By {P2} we can choose some $U_A\in\mathscr U$ that avoids the point $a$; hence $A\not\subseteq U_A$.
```
The previous version did not work because if $A$ does not contain $x$, there is no $U_A\in\mathcal U$ as asserted.

> Clearly, $\{U_A: A \in \mathcal{A}\}$ is a countable family of neighbourhoods of $x$; we show that it is cofinal in $\mathcal{U}$, hence a local base for $x$. But if $U \in \mathcal{U}$, there is $A \in \mathcal{A}$ with $A \subseteq U$, but since $A \not \subseteq U_A$, we cannot have $U \subseteq U_A$. By the hypothesis that $\mathcal{U}$ is linearly ordered by inclusion, we must have $U_A \subset U$ instead.

Reviewer: ```suggestion
Clearly, $\{U_A: A \in \mathcal{A}\}$ is a countable family of neighbourhoods of $x$; we show that it is cofinal in $\mathcal{U}$ with respect to reverse inclusion, hence a local base for $x$.
But if $U \in \mathcal{U}$, there is $A \in \mathcal{A}$ with $A \subseteq U$, but since $A \not \subseteq U_A$, we cannot have $U \subseteq U_A$. By the hypothesis that $\mathcal{U}$ is linearly ordered by inclusion, we must have $U_A \subset U$ instead.
```
specifying cofinal in what sense to avoid ambiguity

---
uid: T000909
if:
  and:
  - P000174: true
  - P000244: true
  - P000135: true
then:
  P000028: true
---

Fix some point $x$; we wish to find a countable local base for $x$. Let $\mathcal{U}$ be a local base for $x$, linearly ordered by inclusion, and $\mathcal{A}$ be a countable local $\pi$-base for $x$. WLOG assume $x$ is not isolated (otherwise $\{\{x\}\}$ is a countable local base for $x$).
Therefore, for any $A \in \mathcal{A}$, there is some $a \in A \setminus \{x\}$.
By {P135} we can choose some $U_A\in\mathscr U$ that avoids the point $a$; hence $A\not\subseteq U_A$.

Clearly, $\{U_A: A \in \mathcal{A}\}$ is a countable family of neighbourhoods of $x$; we show that it is cofinal in $\mathcal{U}$ with respect to reverse inclusion, hence a local base for $x$.

> But if $U \in \mathcal{U}$, there is $A \in \mathcal{A}$ with $A \subseteq U$, but since $A \not \subseteq U_A$, we cannot have $U \subseteq U_A$. By the hypothesis that $\mathcal{U}$ is linearly ordered by inclusion, we must have $U_A \subset U$ instead.

Reviewer: ```suggestion
If $U \in \mathcal{U}$, there is $A \in \mathcal{A}$ with $A \subseteq U$, but since $A \not \subseteq U_A$, we cannot have $U \subseteq U_A$. By the hypothesis that $\mathcal{U}$ is linearly ordered by inclusion, we must have $U_A \subset U$ instead.
```

---
uid: T000909
if:
  and:
  - P000174: true
  - P000244: true
  - P000135: true
then:
  P000028: true
---

Below is a proof that uses {P2} instead of {P135}:

> 

Reviewer: ```suggestion
We first handle the case where $X$ is {P2}.
```

Fix some point $x$; we wish to find a countable local base for $x$. Let $\mathcal{U}$ be a local base for $x$, linearly ordered by inclusion, and $\mathcal{A}$ be a countable local $\pi$-base for $x$. WLOG assume $x$ is not isolated (otherwise $\{\{x\}\}$ is a countable local base for $x$).
Therefore, for any $A \in \mathcal{A}$, there is some $a \in A \setminus \{x\}$.
By {P2} we can choose some $U_A\in\mathscr U$ that avoids the point $a$; hence $A\not\subseteq U_A$.

Clearly, $\{U_A: A \in \mathcal{A}\}$ is a countable family of neighbourhoods of $x$; we show that it is cofinal in $\mathcal{U}$ with respect to reverse inclusion, hence a local base for $x$.
If $U \in \mathcal{U}$, there is $A \in \mathcal{A}$ with $A \subseteq U$, but since $A \not \subseteq U_A$, we cannot have $U \subseteq U_A$. By the hypothesis that $\mathcal{U}$ is linearly ordered by inclusion, we must have $U_A \subset U$ instead.

> By passing to Kolmogorov quotients and appealing to metaproperties, the theorem holds with only {P135}.

Reviewer: ```suggestion
This concludes the proof for the {P2} case.

The general case with {P135} instead of {P2} follows by passing to the Kolmogorov quotient.
```
